src/components/EnterpriseSubsidiesContext/data/hooks.js

```diff
@@ -18,7 +18,7 @@ export const useEnterpriseOffers = ({ enablePortalLearnerCreditManagementScreen,
     const fetchOffers = async () => {
       try {
         const [enterpriseSubsidyResponse, ecommerceApiResponse] = await Promise.all([
-          SubsidyApiService.getSubsidyByCustomerUUID(enterpriseId),
+          SubsidyApiService.getSubsidyByCustomerUUID(enterpriseId, { subsidy_type: 'learner_credit' }),
```

Comment:
nit/suggestion (completely optional, up to you!): if you'd prefer to use camelCase here in the `options`, you might consider something like: `SubsidyApiService.getSubsidyByCustomerUUID(enterpriseId, { subsidyType: 'learner_credit' }),` and then within `SubsidyApiService.getSubsidyByCustomerUUID`:

```js
import { snakeCaseObject } from '@edx/frontend-platform';

const queryParams = new URLSearchParams({
  enterprise_customer_uuid: uuid,
  ...snakeCaseObject(options),
});
```